lib/src/list_multimap/list_multimap_builder.dart

```diff
@@ -12,10 +12,9 @@ part of built_collection.multimap;
 /// [Built Collection library documentation](#built_collection/built_collection)
 /// for the general properties of Built Collections.
 class ListMultimapBuilder<K, V> {
-  bool _copyBeforeWrite;
-  BuiltListMultimap<K, V> _builtListMultimap;
   Map<K, BuiltList<V>> _builtMap;
-  Map<K, ListBuilder<V>> _builderMap = new Map<K, ListBuilder<V>>();
+  BuiltListMultimap<K, V> _builtMapOwner;
```

Comment:
I prefer _builtListMultimap over _builtMapOwner (same for the other Builders).

I think while correct the *Owner bit is a foreign concept that makes it harder to understand the code here.

Why not: setList, setBuiltList instead setSafeList, setListOwner?